apps/airdrop/.env.sample

```diff
@@ -0,0 +1,5 @@
+REACT_APP_GITHUB_CLIENT_ID=Iv1.f2e59042dc8f21b9
+REACT_APP_NAMADA_CHAIN_ID=qc-testnet-5.1.025a61165acd05e
```

Comment:
Should be able to get rid of this one now! `REACT_APP_NAMADA_CHAIN_ID`

Reply:
Yeah yeah better safe than sorry :D